Gemfile.lock

```diff
@@ -268,7 +272,7 @@ GEM
       faraday (>= 0.9)
       sawyer (~> 0.8.0, >= 0.5.3)
     orm_adapter (0.5.0)
-    os_map_ref (0.5.0)
```

Comment:
This is bit unusual. version 0.1.1 is back from 2016. Any particular reasons for downgrading?